kong.conf.default

```diff
@@ -926,6 +926,13 @@
                                         # connection may be kept open
                                         # indefinitely.
 
+#enable_debug_header = true             # Enable the `kong-debug` header function
```

Comment:
We use `on`/`off` for `boolean` configs, not `true`/`false`.

Also @hbagdi , should this be "off" by default?

Reply:
Alternatively, what do we think about adding this to the [headers config option](https://github.com/Kong/kong/blob/6482493ebf3b7ca3b508ab6d00df29322e8c2e94/kong.conf.default#L796-L840) instead? Example:

```
headers = server_tokens, latency_tokens, debug
```